### ISSUE TITLE
Parameters settings not working in trainning.

### ISSUE THREAD

**Issue by 954114865** [bmaltais/kohya_ss#1618]
The parameters I set for trainning(Both Dreambooth and Finetune) don't really be used in trainning, the optimizer, the batch size, the max epoch...nothing is right. I had reinstalled the video driver, installed the old video driver, reinstalled python ,reinstalled several kohya gui version, nothing help.
I did used the gui for a long time, but one day I found it not working properly, I don't know what makes it being like this.
I'm using NVIDIA 3090, AMD Ryzen 3 3700X, 48G RAM, running in SSD, Windows 11.
Here is the running output, you can see the parameters had been read, but not really used in the trainning, instead it gives the adamW and some wired trainning steps I never set:

> 20:30:35-452667 INFO     Start Finetuning...
20:30:35-453668 INFO     ./venv/Scripts/python.exe finetune/merge_captions_to_metadata.py --caption_extension=.txt
                         "G:\Kohya\Training\appletest\2\img" "G:\Kohya\Training\appletest\2/meta_cap.json"
found 3 images.
loading existing metadata: G:\Kohya\Training\appletest\2/meta_cap.json
captions for existing images will be overwritten / 既存の画像のキャプションは上書きされます
merge caption texts to metadata json.
100%|███████████████████████████████████████████████████████████████████████████████████| 3/3 [00:00<00:00, 599.50it/s]
writing metadata: G:\Kohya\Training\appletest\2/meta_cap.json
done!
20:30:42-988535 INFO     ./venv/Scripts/python.exe finetune/prepare_buckets_latents.py
                         "G:\Kohya\Training\appletest\2\img" "G:\Kohya\Training\appletest\2/meta_cap.json"
                         "G:\Kohya\Training\appletest\2/meta_lat.json"
                         "G:/stable-diffusion-webui/models/Stable-diffusion/epicrealism_naturalSinRC1VAE.safetensors"
                         --batch_size=1 --max_resolution=512,512 --min_bucket_reso=64 --max_bucket_reso=512
                         --mixed_precision=bf16
found 3 images.
loading existing metadata: G:\Kohya\Training\appletest\2/meta_cap.json
load VAE: G:/stable-diffusion-webui/models/Stable-diffusion/epicrealism_naturalSinRC1VAE.safetensors
100%|████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:04<00:00,  1.56s/it]
bucket 0 (512, 512): 3
mean ar error: 0.0
writing metadata: G:\Kohya\Training\appletest\2/meta_lat.json
done!
20:30:56-026664 INFO     image_num = 3
20:30:56-028666 INFO     repeats = 3
20:30:56-028666 INFO     max_train_steps = 1
20:30:56-029667 INFO     lr_warmup_steps = 0
20:30:56-030668 INFO     Saving training config to G:\Kohya\Training\appletest\2\2_20231017-203056.json...
20:30:56-044682 INFO     accelerate launch --num_cpu_threads_per_process=2 "./fine_tune.py" --full_bf16
                         --pretrained_model_name_or_path="G:/stable-diffusion-webui/models/Stable-diffusion/epicrealism_
                         naturalSinRC1VAE.safetensors" --in_json="G:\Kohya\Training\appletest\2/meta_cap.json"
                         --train_data_dir="G:\Kohya\Training\appletest\2\img"
                         --output_dir="G:\Kohya\Training\appletest\2\" --logging_dir="G:\Kohya\Training\appletest\2\log"
                         --dataset_repeats=1 --learning_rate=1.0 --enable_bucket --resolution="512,512"
                         --min_bucket_reso=64 --max_bucket_reso=512 --save_model_as=safetensors --output_name="2"
                         --learning_rate="1.0" --lr_scheduler="constant" --train_batch_size="4" --max_train_steps="1"
                         --save_every_n_epochs="50" --mixed_precision="bf16" --save_precision="bf16" --seed="1"
                         --caption_extension=".txt" --cache_latents --optimizer_type="DAdaptAdam" --max_train_epochs=200
                         --max_data_loader_n_workers="0" --bucket_reso_steps=1 --gradient_checkpointing --xformers
                         --bucket_no_upscale --noise_offset=0.0 --sample_sampler=euler_a
                         --sample_prompts="G:\Kohya\Training\appletest\2\sample\prompt.txt" --sample_every_n_epochs="20"
prepare tokenizer
loading existing metadata: G:\Kohya\Training\appletest\2/meta_cap.json
metadata has bucket info, enable bucketing / メタデータにbucket情報があるためbucketを有効にします
using bucket info in metadata / メタデータ内のbucket情報を使います
[Dataset 0]
  batch_size: 1
  resolution: (None, None)
  enable_bucket: True
  min_bucket_reso: None
  max_bucket_reso: None
  bucket_reso_steps: None
  bucket_no_upscale: None
  [Subset 0 of Dataset 0]
    image_dir: "G:\Kohya\Training\appletest\2\img"
    image_count: 3
    num_repeats: 1
    shuffle_caption: False
    keep_tokens: 0
    caption_dropout_rate: 0.0
    caption_dropout_every_n_epoches: 0
    caption_tag_dropout_rate: 0.0
    caption_prefix: None
    caption_suffix: None
    color_aug: False
    flip_aug: False
    face_crop_aug_range: None
    random_crop: False
    token_warmup_min: 1,
    token_warmup_step: 0,
    metadata_file: G:\Kohya\Training\appletest\2/meta_cap.json
[Dataset 0]
loading image sizes.
100%|████████████████████████████████████████████████████████████████████████████████████████████| 3/3 [00:00<?, ?it/s]
make buckets
number of images (including repeats) / 各bucketの画像枚数（繰り返し回数を含む）
bucket 0: resolution (512, 512), count: 3
mean ar error (without repeats): 0.0
prepare accelerator
loading model for process 0/1
load StableDiffusion checkpoint: G:/stable-diffusion-webui/models/Stable-diffusion/epicrealism_naturalSinRC1VAE.safetensors
UNet2DConditionModel: 64, 8, 768, False, False
loading u-net: <All keys matched successfully>
loading vae: <All keys matched successfully>
loading text encoder: <All keys matched successfully>
Disable Diffusers' xformers
prepare optimizer, data loader etc.
use AdamW optimizer | {}
running training / 学習開始
  num examples / サンプル数: 3
  num batches per epoch / 1epochのバッチ数: 3
  num epochs / epoch数: 534
  batch size per device / バッチサイズ: 1
  total train batch size (with parallel & distributed & accumulation) / 総バッチサイズ（並列学習、勾配合計含む）: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 1600
steps:   0%|                                                                                  | 0/1600 [00:00<?, ?it/s]
epoch 1/534
steps:   0%|                                                            | 3/1600 [01:02<9:17:15, 20.94s/it, loss=0.032]
epoch 2/534
steps:   0%|▏                                                          | 6/1600 [02:02<9:02:20, 20.41s/it, loss=0.0173]
epoch 3/534
20:33:22-634398 INFO     The running process has been terminated.

**Comment by andrewtvuong** [comment]
I have this same issue

**Comment by 954114865** [comment]
I solved this issue by windows repair install.

**Comment by 954114865** [comment]
I solved this issue by windows repair install.
